test/integration/start-integration-env.sh

```diff
@@ -100,10 +100,6 @@ yarn --cwd $BASEDIR/smart-contracts peggy:lock ${ADDR} 0x00000000000000000000000
 export USER1ADDR=$(cat $NETDEF | yq r - "[1].address")
 echo "export USER1ADDR=$USER1ADDR" >> $envexportfile
 
-sleep 5
-yarn --cwd $BASEDIR/smart-contracts peggy:lock ${USER1ADDR} 0x0000000000000000000000000000000000000000 $(to_wei 10)
-sleep 5
-
```

Comment:
Here's where we'd want to use the generateNBlocks yarn command from a shell script

Reply:
Agree, will optimize the script.

Reply:
We've added these tests